src/components/cards/InformationCard.tsx

```diff
@@ -25,13 +25,19 @@ const InformationCard = ({ data }: any) => {
           data.Extra
             ? [
                 ...data.PARSED_RELATES_TO.map((item: any) => {
+                  let newItem;
+                  if (item.split(" ").length > 1) {
+                    newItem = item.split(" ")[1];
+                  } else {
+                    newItem = item;
+                  }
                   return {
-                    targetId: item,
+                    targetId: newItem,
```

Comment:
Do we need this component?

Reply:
this one is still being used on caseStudy and coreToolKits that is the next to be refactored by using new library react flow